nixos/modules/services/logging/logrotate.nix

```
@@ -240,11 +240,12 @@ in
   };
 
   config = lib.mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.logrotate ];
```

```suggestion
    environment.systemPackages = [ (lib.getMan pkgs.logrotate) ];
```

not sure if we should od that or not

---

You can [run logrotate manually](https://wiki.archlinux.org/title/Logrotate#Usage), although it's a rare use case but I think maybe we should keep this.